components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1656,25 +1618,19 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
                      call get_curr_date(year,month,day,tod)
                      !populate with corresponding pbuf
                      !variables
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_1_idx, aer_tau_bnd_lw_mon_1)
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_2_idx, aer_tau_bnd_lw_mon_2)
-                     if (current_month .ne. month) then
-                        aer_tau_bnd_lw_mon_1 = aer_tau_bnd_lw_mon_2
-                        if (month==12) then
+                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_idx, aer_tau_bnd_lw_mon)
+                     if (month==12) then
```

Comment:
current month and next month were already defined above in this same function, right? I suspect these definitions aren't needed (though they're cheap and perhaps nice to remind people where these variables were defined?)

Reply:
The above definition appears to be nested within an `if` statement.  Hence the repeat.  However, since the same action is taken regardless of the result of the `if` statement we might as well move this chunk of code to just before the `if` statement.